.changeset/fuzzy-spoons-rush.md

```diff
@@ -0,0 +1,7 @@
+---
+'skuba': patch
+---
+
+deps: Prettier 3.0.3
```

Comment:
This bump up brought some slight formatting changes so I thought I'd call it out.